intuita-webview/src/codemodList/CodemodArguments/FormField/index.tsx

```diff
@@ -18,7 +18,8 @@ const FormField = (props: Props) => {
 			<VSCodeTextField
 				placeholder={name}
 				value={String(value)}
-				onInput={onChange}
+				// @ts-expect-error value exists on target
```

Comment:
I would not use `@ts-expect-error` with complex ASTs, this directive disables the checks for the entire line (it's not targeting a specific issue).
I'd rather check the event for the target value and execute `onChange` only if the latter exists.